modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -387,6 +387,7 @@ TEST_P(Test_ONNX_layers, PReLU)
 TEST_P(Test_ONNX_layers, Clip)
 {
     testONNXModels("clip", npy);
+    testONNXModels("clip_init_min_max");
 }
```

Comment:
Please create separate test for each model (grouping them is not a good practice).

```
TEST_P(Test_ONNX_layers, Clip_init_min_max)
{
    testONNXModels("clip_init_min_max");
}
```

Reply:
Thanks, I will update the code in the next commit.